common/djangoapps/course_modes/migrations/0003_auto_20151113_1443.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('course_modes', '0002_coursemode_expiration_datetime_is_explicit'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='coursemode',
+            name='expiration_datetime_is_explicit',
+            field=models.BooleanField(default=False),
```

Comment:
Correct.  There are two commits here.  https://github.com/edx/edx-platform/commit/e390c89947e82b18af0f71ec0b8117135fbde98c Will need to go out first (potentially outside of the typical release cycle).

Then https://github.com/edx/edx-platform/commit/a9e31cedfef0cac1a74f58cc9636871ce6185f8a will go out after (through the standard release process)